modules/installation-configuration-parameters.adoc

```diff
@@ -188,6 +188,10 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-installer-provisioned-vsphere"]
 :vsphere:
 endif::[]
+ifeval::["{context}" == "installing-vsphere-installer-provisioned-zones"]
+:vsphere:
+:vsphere-zones:
+endif::[]
```

Comment:
Make sure you add a corresponding block at the end of the file that unsets these variables:
```
ifeval::["{context}" == "installing-vsphere-installer-provisioned-zones"]
:!vsphere:
:!vsphere-zones:
endif::[]
```